src/src/tx_main.cpp

```diff
@@ -897,6 +897,11 @@ static void setupTarget()
     pinMode(GPIO_PIN_UART1TX_INVERT, OUTPUT); // TX1 inverter used for debug
     digitalWrite(GPIO_PIN_UART1TX_INVERT, LOW);
 #endif
+
+//#if defined(TARGET_TX_FLYSKY_FRM301)
+//  Serial.setTx(PA2);
+//  Serial.setRx(PA3);
+//#endif
```

Comment:
Any reason to keep thee comments?

Reply:
I think these should be enabled and pins configured properly to make the UART working :) 
note this @porfiheli